src/TrackPanel.cpp

```diff
@@ -7195,9 +7195,9 @@ void TrackPanel::DrawTracks(wxDC * dc)
 ///  - Zooming Indicators.
 ///  - Fills in space below the tracks.
 void TrackPanel::DrawEverythingElse(wxDC * dc,
-                                    const wxRegion region,
-                                    const wxRect WXUNUSED(panelRect),
```

Comment:
We should probably just drop this parameter altogether since it's not used anyway.